merge-script-assignment2.sh

```diff
@@ -1,28 +1,32 @@
-#1/bin/sh
+#!/bin/sh
 
 A2_URL=$1
 
 if [ $# -lt 1 ]
-	then
-		echo "Missing A2 URL"
-                echo "Usage: [./merge-script-assignemtn2.sh] [your_A2_Github_SSH_URL]"
+        then
+                echo "Usage: $0 [your_A2_Github_SSH_URL]"
                 exit 1
-	else
-                echo "A2 URL USED"
+        else
+                echo "Using assignment2 repo at $1"
 fi
 
+# Repoint origin to the student's assignemnt 2 repo
```

Comment:
I'm conflicted about this script generally because it makes some assumptions that may not be true in all cases; when they're not true, they could cause unwanted side-effects.